src/ui/utils/timeline.ts

```diff
@@ -309,3 +310,48 @@ export function filterToFocusRegion<T extends TimeStampedPoint>(
 
   return sortedPoints.slice(beginIndex, endIndex);
 }
+
+function assertSorted(a: TimeStampedPoint[]) {
+  a.reduce(
+    (prev, curr) => {
+      assert(prev.time <= curr.time);
+      return curr;
+    },
+    { point: "0", time: 0 }
+  );
+}
```

Comment:
I'm seeing this assertion throw errors for recording 436b7cf5-50f1-4e69-894d-a61343e128c2 but it's not really clear how to use that information.

Reply:
I _think_ this assert actually uncovered a bad recording (see BAC-2339).

I'm going to change it for now though to (1) only run in DEV and (2) compare points rather than time (since those are more fine-grained anyway). Let's chat if you think this is a bad move!